functions/divide.h

```diff
@@ -0,0 +1,58 @@
+#ifndef FUNCTIONS_DIVIDE_H
+#define FUNCTIONS_DIVIDE_H
+
+// Usage: Divide<F, V>
+// Divide F by V
+// If V = 0, returns 0
+// F, V: FUNCTION, 
+// return value: FUNCTION
+
+class BladeBase;
+
+template<class F, class V>
+class DivideBase {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    v_.run(blade);
+  }
+  int getInteger(int led) {
+    int v = v_.getInteger(led);
+    if (v == 0) {
+      return 0;
+    } else {
+      return f_.getInteger(led) / v;
```

Comment:
I wonder if we want this to be f * 32768 / v  ?
If it's used mostly for fixed-point math where the values are between 0 and 32768, that would make more sense.
If it's used for other things, then it would not.

Reply:
The original intent is not fixed-point, the uses I have been testing are to divide one function by another to get the actual value.